scss/components/brand-lockup/_brand-lockup.scss

```diff
@@ -0,0 +1,98 @@
+@charset 'UTF-8';
+
+// My Config.
+@import "config/config";
+
+// Decanter config.
+@import "decanter/core/utilities/decanter-utilities";
+
+$logo-width: 110px;
+
+.block--lockup {
+  background: transparent url('/themes/stanford/matson/img/source/bg-logo.png') no-repeat center center;
+  background-size: 160px 140px;
+  position: relative;
+  margin: 0 auto 0;
+  padding-top: 14px;
+  padding-bottom: 14px;
+  width: 160px;
+  z-index: 4;
+
+  @include grid-media($header-md) {
```

Comment:
It looks like it is used in a few places and slots between medium and large. We can add this to the default grid containers so that it is available everywhere.